### PR TITLE
Add Map Roulette Upload to AWS script

### PR DESCRIPTION
**Pull request by atiannicelli**
### Description:

Add a bit of code to the AWS script so Map Roulette challenges can be created from data produced from the AWS check script.

### Potential Impact:

None to existing functionality. Adds a new command to the script.

### Unit Test Approach:

I ran this a few times against different log files to produce MR projects and challenges.
I found one issue during testing for this change: https://github.com/osmlab/atlas-checks/issues/415
Once that bug is fixed processing mr project will be much more consistent. 

### Test Results:

All good.

### CODE REVIEW

**Review by sayas01 — APPROVED** [review]
LGTM!

File: scripts/cloud-check-control/cloudAtlasCheckControl.py [hunk 1]
```diff
@@ -231,6 +244,55 @@ def sync(self):
...
+
+        # push output to s3
+        cmd = "aws s3 sync --only-show-errors --exclude *.crc {} s3://{} ".format(
+            self.atlasOutDir, self.s3OutFolder
+        )
+        if self.ssh_cmd(cmd):
+            finish("Unable to sync with S3", -1)
+
+        cmd = (
+            "java -cp atlas-checks/build/libs/atlas-checks-*-SNAPSHOT-shaded.jar"
```
Comment (seancoulter) [inline scripts/cloud-check-control/cloudAtlasCheckControl.py:278]: Could we make the jar location configurable?

**Review by seancoulter — COMMENTED** [review]
Looks good, excited for this.

File: scripts/cloud-check-control/cloudAtlasCheckControl.py (hunk 1 above)
Comment (atiannicelli) [reply scripts/cloud-check-control/cloudAtlasCheckControl.py:278]: Sure.

**Review by seancoulter — APPROVED** [review]

**Review by sayas01 — APPROVED** [review]

**Pull request merged**